app/main.py

```diff
@@ -1 +1,64 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(
+            self,
+            name: str
+    ) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(
+            self,
+            name: str
+    ) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(
+            self,
+            name: str
+    ) -> None:
+        super().__init__(name)
+        self.skills = ["Python", "SQL", "Django"]
```

Comment:
Use the extend() method or += operator to add the necessary skills to the list inherited from the parent class